### PR TITLE
[FEATURE] add isPrime

### PR DESCRIPTION
**Pull request by fejes713**
## add isPrime
Added a function that checks if the provided number is a prime number.

> A prime number is a natural number greater than 1 that has no positive divisors other than 1 and itself.

## Types of changes
- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## Checklist:

- [x] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [x] I have checked that there isn't any PR doing the same
- [x] I have read the **CONTRIBUTING** document.

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1216,6 +1217,23 @@ const isEven = num => num % 2 === 0;
 
 [⬆ back to top](#table-of-contents)
 
+### isPrime
+
+Checks if the provided intiger is prime number.
```
Comment (Pl4gue) [inline README.md]: Please fix this typo and the grammar issue as well:
"Checks if the provided integer is a prime number."

File: docs/index.html [hunk 2]
```diff
@@ -761,6 +762,16 @@ <h3>Utility
 <pre><code class="language-js">const isEven = num =&gt; num % 2 === 0;
 // isEven(3) -&gt; false
 </code></pre>
+</div></div><br/><div class="card fluid"><div class="section double-padded"><h3 id="isprime">isPrime</h3></div><div class="section double-padded">
+<p>Checks if the provided intiger is prime number.</p>
```
Comment (Pl4gue) [inline docs/index.html]: Here as well. (s.a.)

File: snippets/isPrime.md [hunk 3]
```diff
@@ -0,0 +1,14 @@
+### isPrime
+
+Checks if the provided intiger is prime number.
```
Comment (Pl4gue) [inline snippets/isPrime.md]: Here as well. (s.a.)

**Comment by Pl4gue** [comment]
Also, I remind you of the new PR rules: this is a new Feature, so please add the [FEATURE] tag as prefix in the PR title :smile:

File: snippets/isPrime.md (hunk 3 above)
Comment (fejes713) [reply snippets/isPrime.md]: Thanks @Pl4gue! Appreciate it!

**Comment by atomiks** [comment]
You forgot the curly braces?

Also, can't you divide the number by 2 so it runs twice as fast for primes?

```js
const isPrime = num => {
  for (let i = 2; i < num / 2; i++) if (num % i === 0) return false;
  return num >= 2;
};
```

**Pull request merged**